src/docs/tool-index.dox

```diff
@@ -35,6 +35,13 @@ Why is my CPU busy? FlameGraphs help visually summarize on-CPU call stacks and
 allow for the quick identification of hot code paths. Here we explain how to
 generate FlameGraphs from WiredTiger `perf` data.
 
+@subpage tool-libfuzzer
+
+LLVM LibFuzzer is in-process, coverage-guided, evolutionary fuzzing engine. It feeds a series of
+fuzzed inputs via a "target" function and attempts to trigger crashes, memory bugs and undefined
+behavior. Here we explain how to build and run existing fuzz targets, implement new ones and
```

Comment:
General comment: It is in this file a few times, and in the new file a lot - we try to avoid personal pronouns in the documentation. So avoid "we", "you", "our", etc and instead use general terms like `WiredTiger` or `users`.

Reply:
d70b1a0